sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/Models/ArtifactManifestProperties.cs

```diff
@@ -72,8 +72,6 @@ internal ArtifactManifestProperties(string registryLoginServer, string repositor
         public string RepositoryName { get; }
         /// <summary> Manifest. </summary>
         public string Digest { get; }
-        /// <summary> Image size. </summary>
-        public long? Size { get; }
```

Comment:
nit: should this change to SizeInBytes, or did I just miss seeing that elsewhere?

Reply:
Yes, this has been renamed to `SizeInBytes`. So it has been renamed from here.